src/octoprint/plugins/logging/static/js/logging.js

```diff
@@ -36,7 +37,7 @@ $(function() {
         );
 
         self.requestData = function() {
-            OctoPrint.logs.list()
+            OctoPrint.get(logsURL)
```

Comment:
This should still use the js client library, albeit at its new location `OctoPrint.plugins.logs.*` (see another review comment on that).